packages/reports/tests/integration/components/filter-values/multi-value-input-test.ts

```diff
@@ -0,0 +1,85 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import $ from 'jquery';
+import { render, fillIn, triggerEvent, click } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+import FragmentFactory from 'navi-core/services/fragment-factory';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import MultiValueInput from 'navi-reports/components/filter-values/multi-value-input';
+
+type ComponentArgs = MultiValueInput['args'];
+interface TestContext extends Context, ComponentArgs {}
+
+module('Integration | Component | filter values/multi value input', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function(this: TestContext) {
+    const fragmentFactory = this.owner.lookup('service:fragment-factory') as FragmentFactory;
+    this.filter = fragmentFactory.createFilter('dimension', 'bardOne', 'age', { field: 'id' }, 'in', ['10']);
+    this.onUpdateFilter = () => null;
+
+    await render(hbs`<FilterValues::MultiValueInput
+            @filter={{this.filter}}
+            @onUpdateFilter={{this.onUpdateFilter}}
+            @isCollapsed={{this.isCollapsed}}
+        />`);
+  });
+
+  test('it renders', function(this: TestContext, assert) {
+    assert.expect(1);
+
+    assert.equal(
+      $('.emberTagInput-tag')[0].innerText.trim(),
+      this.filter.values[0],
+      'The value select contains an input with the first filter value as a tag'
+    );
```

Comment:
```suggestion
    assert.dom('.emberTagInput-tag').hasText(
      this.filter.values[0],
      'The value select contains an input with the first filter value as a tag'
    );
```
Can we cleanup the jquery usage in this test?